di-ipv-core-stub/src/main/java/uk/gov/di/ipv/stub/core/utils/HandlerHelper.java

```diff
@@ -393,4 +397,19 @@ private RSAKey getEncryptionPublicKey(CredentialIssuer credentialIssuer)
                 new String(
                         Base64.getDecoder().decode(credentialIssuer.publicEncryptionJwkBase64())));
     }
+
+    public boolean isSignedUsingECDSA(SignedJWT signedJWT) throws JOSEException {
```

Comment:
To adhere to the single responsibility principle this method should be split into 2 methods:
- `void checkSignatureFormat(SignedJWT signedJWT)` this would contain the code on lines 402-406
- `boolean verifySignedJwt(SignedJWT signedJWT)` this would contain the code on lines 408-413